src/FormLabel/index.js.flow

```diff
@@ -4,6 +4,7 @@ export type Props = {
   children: React$Node,
   filled?: boolean,
   disabled?: boolean,
+  required?: boolean,
 };
```

Comment:
Should be typing for `FormLabel` strict too?

Reply:
Because when someone will import `FormLabel` when needed to use `compact InputField`, then it's necessary to be  strict in flow type, otherwise the person will be able to pass eg. `className`.

Reply:
I think if someone want to use FormLabel they should read documentation firstly how to use. If there will be more questions about this, we can update, so for now it is OK.